### DISCUSSION TITLE
Open DTU mit Shelly 3em steuern

### DISCUSSION THREAD

**Discussion by HB2907** [tbnobody/OpenDTU#1062] [Ideas]
Hallo,

Wäre es möglich die Leistung des WR mit den Daten vom Shelly 3em zu steuern. Es existiert eine Lösung auf github aber dazu wird ein Raspi benötigt der sehr teuer ist und viel Strom verbraucht.
Klausli hat sowas programmiert für den Soyosource Inverter.

Danke und gute Zeit

**Comment by CBFOUR** [comment]
Moin
https://github.com/helgeerbe/OpenDTU-OnBattery
Das schau dir mal an, ist eine Erweiterung von diesem Project.

Cheers

**Comment by HB2907** [comment]
Super Danke, das ist es was ich gesucht habe. Leider ist das für mich zu
komliziert.

Am Do., 22. Juni 2023 um 09:47 Uhr schrieb CBFOUR ***@***.***

**Reply by CBFOUR** [reply]
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/234
HTTP String und JSON Pfad beim Shelly 3 em

Cheers

**Comment by HB2907** [comment]
Hi, danke für deine Mühe aber ich könnte nur mit einem fertigen .bin was
anfangen das ich nur flashen muss. für alles andere bin ich zu doof.

Am Do., 22. Juni 2023 um 12:02 Uhr schrieb CBFOUR ***@***.***
